### PR TITLE
Add postprocessing support to GCP

### PR DESCRIPTION
**Pull request by lathanh**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -39,6 +39,7 @@
         'dask[complete]>=2022.10.0',
         'docker',
         's3fs[boto3]',
+        'gcsfs',
```
Comment (nweires) [inline setup.py]: This already on the list, on line 47.

File: buildstockbatch/gcp/gcp.py [hunk 2]
```diff
@@ -749,6 +758,69 @@ def run_task(cls, task_index, job_name, gcs_bucket, gcs_prefix):
...
+
+        :returns: A `GCSFileSystem()`.
+        """
+        return GCSFileSystem()
+
+    def process_results(self, skip_combine=False, use_dask_cluster=True):
+        """
+        Overrides `BuildStockBatchBase.process_results()`.
+
+        While the BSB implementation uploads to S3, this uploads to GCP Cloud Storage. The BSB
```
Comment (nweires) [inline buildstockbatch/gcp/gcp.py]: What is the "BSB implementation" here? Is it referring to the base class implementation?

**Review by nweires — APPROVED** [review]

File: setup.py (hunk 1 above)
Comment (lathanh) [reply setup.py]: Done.

File: buildstockbatch/gcp/gcp.py (hunk 2 above)
Comment (lathanh) [reply buildstockbatch/gcp/gcp.py]: Yes, the base class. I've changed "BSB" to "BuildStockBatchBase"

**Pull request merged**